Makefile

```diff
@@ -98,5 +100,12 @@ test-rc-deps: ## Test providers RC by building an image with given dependencies
 shell:  ## Runs a shell within a container (Allows interactive session)
 	docker compose -f dev/docker-compose.yaml run --rm airflow-scheduler bash
 
+bump-version:  ## Bump versions in files locally. By default bump to DEV. set env ASTRO_PROVIDER_VERSION to change default version (1.0.0 | 1.1.0 | 1.1.1).
```

Comment:
maybe we'll need to change here as well?

Reply:
yep, thanks for pointing it out.